cdap-ui/app/cdap/components/DataPrep/DataPrepTable/index.js

```diff
@@ -273,7 +286,12 @@ export default class DataPrepTable extends Component {
                   >
                     <DataQuality columnInfo={this.state.columns[head.name]} />
                     <div className="column-wrapper-container">
-                      <div className="col-type">{captialize(types[head.name] || 'unknown')}</div>
+                      <div className={classnames("col-type", {
+                        'data-type-updated': this.state.highlightedDataType === head.name
+                        })}
+                      >
+                        {captialize(types[head.name] || 'unknown')}
```

Comment:
nit: i18n?

Reply:
this is just moved from above, but sure.